### PR TITLE
release/v6.2.0

### PR DESCRIPTION
**Pull request by ipfaze**
- release(v6.2.0): bump version to v6.2.0

- [ ] Add a [changelog entry](https://changelog.scalingo.com/)
related to https://github.com/Scalingo/documentation/issues/1937

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## To Be Released
+
+## 6.2.0
```
Comment (EtienneM) [inline CHANGELOG.md:5]:
```suggestion
## 6.2.0

```

**Review by EtienneM — COMMENTED** [review]
Do we eventually settle on using "too many requests" error for the work you are doing?

**Comment by ipfaze** [comment]
@EtienneM 
We hadn't decided on the question yet, do you have any other suggestions?

**Comment by EtienneM** [comment]
> @EtienneM We hadn't decided on the question yet, do you have any other suggestions?

If our API never return "too many requests", my suggestion is to not implement it in go-scalingo.

**Comment by ipfaze** [comment]
@EtienneM 
The only error closed to our need is: 422 Unprocessable Entity
Do you think it stick with our use case?

**Review by EtienneM — APPROVED** [review]

**Pull request merged**